nautobot/extras/api/serializers.py

```diff
@@ -128,8 +129,11 @@ def get_field_names(self, declared_fields, info):
 
     @extend_schema_field(serializers.URLField())
     def get_notes_url(self, instance):
-        notes_url = get_route_for_model(instance, "notes", api=True)
-        return reverse(notes_url, args=[instance.id], request=self.context["request"])
+        try:
+            notes_url = get_route_for_model(instance, "notes", api=True)
+            return reverse(notes_url, args=[instance.id], request=self.context["request"])
+        except NoReverseMatch:
+            return "Notes feature is not available for this model. Please implement it before including NotesSerializerMixin"
```

Comment:
Seems weird to me to either return a usable URL or else return a human-readable string? If anything, would it be better to *log* the human-readable message and return None?